Src/java/quick/src/main/resources/org/hl7/fhir/fhir-modelinfo-4.0.0.xml

```diff
@@ -554,7 +554,7 @@
         <ns4:element name="name" elementType="FHIR.string"/>
         <ns4:element name="type" elementType="FHIR.DeviceNameType"/>
     </ns4:typeInfo>
-    <ns4:typeInfo xsi:type="ns4:ClassInfo" name="MeasureReport" retrievable="true" baseType="FHIR.DomainResource">
+    <ns4:typeInfo xsi:type="ns4:ClassInfo" name="MeasureReport" retrievable="true" baseType="FHIR.DomainResource" primaryCodePath="measure.topic">
```

Comment:
Similar to above -- there is magic happening here.  Strictly speaking, `measure` is just a URL.  The notion that the URL should be resolved to an instance before getting the `topic` property requires special knowledge, that I think is undocumented by the CQL spec.  If we need to do this, I really think we should consider using FHIRPath -- which I assume supports resolving `canonical`. (FHIR 4.0.0 `MeasureReport`)

Reply:
Agreed, leaving out for now